projects/storefrontlib/src/lib/my-account/address-book/address-book.component.ts

```diff
@@ -89,7 +89,7 @@ export class AddressBookComponent implements OnInit, OnDestroy {
     return this.actions.subscribe(action => {
       switch (action.type) {
         case LOAD_USER_ADDRESSES_SUCCESS: {
-          this.userService.addressesState$.subscribe(data => {
+          this.userService.getAddressesState().subscribe(data => {
```

Comment:
Can we use `getAddresses()` instead of exposing the whole user state?

Reply:
Yes, here we can. I replaced that.